model/flow/chunk_test.go

```diff
@@ -104,3 +106,52 @@ func TestChunkList_Indices(t *testing.T) {
 		require.Contains(t, indices, uint64(0), uint64(2), uint64(4))
 	})
 }
+
+func TestChunkIndexIsSet(t *testing.T) {
```

Comment:
These tests, which moved from the ingestion engine tests, should belong to this file